docs/installation_configuration_guide/ModelConfiguration.md

```diff
@@ -21,7 +21,7 @@ This part of configuration defines how to configure the executor (e.g. Flink job
 | checkpointConfig.minPauseBetweenCheckpoints        | Low        | duration | checkpointInterval / 2 | [Minimal pause](https://ci.apache.org/projects/flink/flink-docs-stable/docs/deployment/config/#execution-checkpointing-min-pause) between checkpoints                         |
 | checkpointConfig.maxConcurrentCheckpoints          | Low        | int      | 1                      | [Maximum concurrent checkpoints](https://ci.apache.org/projects/flink/flink-docs-stable/docs/deployment/config/#execution-checkpointing-max-concurrent-checkpoints) setting   |
 | checkpointConfig.tolerableCheckpointFailureNumber  | Low        | int      |                        | [Tolerable failed checkpoint](https://ci.apache.org/projects/flink/flink-docs-stable/docs/deployment/config/#execution-checkpointing-tolerable-failed-checkpoints) setting    |
-| rocksDB.checkpointDataUri                          | High       | string   |                        | URI for storing checkpoints. Note: This **has** to be accessible from Nussknacker designer (for snapshot validation) **and** from Flink cluster (jobmanager and taskmanagers) |
+| rocksDB.enable                                     | Medium     | boolean  | true                   | Enable RocksDB state backend support
```

Comment:
There is also mention in Operations, let's change it also there

Reply:
Was typo there and because of that, I've missed that - changed.